kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -23,16 +23,26 @@ The script assumes that either the image, `container-registry.oracle.com/middlew
 
 ```
 $ ./create-rcu-schema.sh -h
-usage: ./create-rcu-schema.sh -s <schemaPrefix> -t <rcuType> -d <dburl> -i <image> -s <dockerstore> [-h]
+usage: ./create-rcu-schema.sh -s <schemaPrefix> -t <schemaType> -d <dburl> -i <image> -u <imagePullPolicy> -p <docker-store> -n <namespace> -q <sysPassword> -r <schemaPassword>  -o <rcuOutputDir>  [-h]
```

Comment:
A tiny one
-p <ImagePullSecret>

Reply:
-p <ImagePullSecret>

Reply:
Please confirm what changes required. The original README.md was not inline with actual "./create-rcu-schema.sh -h"  output and was pointing -s twice for schemaPrefix and dockerstore. 
The PR has README.md changes with output of "./create-rcu-schema.sh -h" from actual output.

Reply:
I think the README.md should reflect the real output of the latest change of ./create-rcu-schema.sh.

Reply:
Currently, README.md has real output of "./create-rcu-schema.sh -h" and "./drop-rcu-schema.sh -h".
Please confirm if the real output of "./create-rcu-schema.sh -s domain1" and "./drop-rcu-schema.sh -s domain1" also need to be updated in README.md.

Reply:
Yes, please. 
Thank you.

Reply:
Please review the latest commit with the required updates into README.md.
Thanks,
Asha